prow/gerrit/fakegerrit/fakegerrit.go

```diff
@@ -80,6 +86,23 @@ func (fg *FakeGerrit) GetComments(id string) map[string][]*gerrit.CommentInfo {
 	return nil
 }
 
+func (fg *FakeGerrit) AddChangeAfter(projectName string, change *gerrit.ChangeInfo) {
+	fg.lastUpdateTime = makeTimeStamp(fg.lastUpdateTime.Add(time.Hour * 12).UTC())
```

Comment:
Why can't we just add 12 hours to the current time (instead of the lastUpdateTime)? Isn't that simpler (and avoid having to keep track of lastUpdateTime in the FakeGerrit struct)? Maybe I'm missing something.

Reply:
I think there might be a precision problem if we just add 12 hours to the current time, for example in a test:
1. current time: 00:00:00:000, then lastUpdateTime is 12:00:00:000,
2. current time: 00:00:00:001, then lastUpdateTime is 12:00:00:001

and the time comparison might not be accurate enough to tell these two values are different. (just my guess, @mpherman2  might have a better reason)